tests/build_memory_index.cpp

```diff
@@ -48,18 +51,21 @@ int build_in_memory_index(const diskann::Metric& metric,
   if (label_file == "") {
     index.build(data_path.c_str(), data_num, paras);
   } else {
+    convert_labels_string_to_int(label_file, labels_file_to_use, 
+                                  mem_labels_int_map_file, universal_label);
     if (universal_label != "") {
       LabelT unv_label_as_num = std::stoul(universal_label);
```

Comment:
Can we assume universal_label is the string representation of a number here, or do we have to use the get_converted_label function here instead?

Reply:
We can use get_converted_label function here